File: test/export/vb/05-para.vb
Module SIN_TITULO
	Sub Main()
		Dim c As Double
		Dim i As Integer
		Dim j As Integer
		Dim a(10) As Double
		For i=1 To 10
			a(i) = i*10
		Next i
		For Each elemento In a
			Console.WriteLine(elemento)
		Next
		Dim b(3,6) As Double
		c = 0
		For Each x In b
			c = c+1
			x = c
		Next
		For i=3 To 1 Step -1
			For j=1 To 5 Step 2
				Console.WriteLine(b(i,j))
			Next j
		Next i
	End Sub
End Module

File: test/export/cs/05-para.cs
using System;
namespace PSeInt {
	class sin_titulo {
		static void Main(string[] args) {
			double c;
			int i;
			int j;
			double[] a = new double[10];
			for (i=1;i<=10;i++) {
				a[i-1] = i*10;
			}
			foreach (int elemento in a) {
				Console.WriteLine(elemento);
			}
			double[,] b = new double[3,6];
			c = 0;
			foreach (int x in b) {
				c = c + 1;
				// No se puede asignar el valor de c en x ya que x es la variable de iteracion 
				//x = c;
			}
			for (i=3;i>=1;i--) {
				for (j=1;j<=5;j+=2) {
					Console.WriteLine(b[i-1,j-1]);
				}
			}
		}
	}
}
